docs/SOPs/data_retraction_SOP.md

```diff
@@ -88,6 +109,22 @@ Currently, we store contributor data in the cloud **before** and **while** broke
     **Example**: aws s3 rm -r s3://org-hca-data-archive-upload-prod/<span style="color:red">779ecf45-e930-459e-b87b-c89d3c4546c7/</span>
     
     <span style="color:red">**IMPORTANT NOTE**</span>: Please ensure the path you introduced points out to the specific area and not to the bucket, as you might risk deletion of all the upload areas otherwise. The upload area is highlighted in red in the example.
+
+**Prabhat: THIS IS VERY RISKY**
+A better way to do this is to send a DELETE request to the Upload Service as following (for dev):
+
+```
+# retrieve Upload Service Api-Key (dev)
+aws --region us-east-1 secretsmanager get-secret-value --secret-id ingest/dev/secrets --query SecretString --output text | jq -jr .staging_api_key
+
+# delete an upload area (dev)
+curl -X DELETE "https://upload.dev.archive.data.humancellatlas.org/v1/area/<upload_area_uuid>" -H  "accept: application/json" -H  "Api-Key: <API-KEY>"
+```
+
+Also, should we not make sure to clean up data in upload areas in other pre-prod environments -- dev, and staging? Data often remains there and never gets cleaned up...
```

Comment:
Good point, I will add a note about other environments. 

Lately we have not been testing in staging so there shouldn't be any problem usually but it should be included somewhere that other environments should be at least checked